src/main/java/com/graphhopper/converter/api/GisgraphySearchEntry.java

```diff
@@ -0,0 +1,224 @@
+package com.graphhopper.converter.api;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.graphhopper.converter.data.CountryInfo;
+
+/**
+ * @author David Masclet
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class GisgraphySearchEntry {
+
+
+	private static List<String> NAME_HOUSE_COUNTRYCODE = new ArrayList<String>() {
+		{
+			add("DE");
+			add("BE");
+			add("HR");
+			add("IS");
+			add("LV");
+			add("NL");
+			add("NO");
+			add("NZ");
+			add("PL");
+			add("RU");
+			add("SI");
+			add("SK");
+			add("SW");
+			add("TR");
+		}
+	};
+
+
+	
+
+	@JsonProperty("feature_id")
+	private long featureId;
```

Comment:
Why featureId, if we don't use it?

I have seen, that for `https://services.gisgraphy.com/fulltext/fulltextsearch` there is a `openstreetmap_id`, which might be more helpful, because we return OSM id in the response.

For Geocoding I have seen that there is the sourceId, but it's not guaranteed that it's the OSM id? Would it make sense to somehow show the end user of the Gisgraphy service where the results come from, I opened a separate issue for this [here](https://github.com/gisgraphy/gisgraphy/issues/23).

Reply:
we never use it, but I use the same class in Gisgraphy and in geocoder-converter. that's why there is maybe some fields not used here.